invenio_vocabularies/records/dumper_extensions.py

```diff
@@ -27,4 +27,8 @@ def dump(self, record, data):
 
     def load(self, data, record_cls):
         """Load data."""
-        data.pop("vocabulary_type", None)
+        vocabulary_type_id = data.get("vocabulary_type_id")
```

Comment:
Minor: This should not be added back. I removed it because the dump/load has to produce the exact same record:

```python
rec = Record.get_record(...)
assert rec == Records.loads(rec.dumps())
```

Thus if you add a new property in the dump, it has to be removed again in the load method. This is to ensure that we don't have different record representations when we're getting a record from database or elasticsearch.

Reply:
Thanks for the explanation, I will fix it differently.